deepquest/__main__.py

```diff
@@ -33,6 +37,17 @@ def score(args):
     from . import score
     score(args.files)
 
+def set_gpu_id(gpuid):
+    import os
+    os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
+    gpustr = ''
+    for g in gpuid:
+        gpustr += str(g) + ','
+
+    os.environ["CUDA_VISIBLE_DEVICES"] = gpustr[0:-1]
+    print(os.environ["CUDA_VISIBLE_DEVICES"])
+    import sys
+    sys.exit()
```

Comment:
I don't get these two lines, what are they for in this function?

Reply:
Woops these are left in from debugging, will remove. Thanks.